velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -718,6 +718,18 @@ std::pair<std::unique_ptr<TaskCursor>, std::vector<RowVectorPtr>> readCursor(
   return {std::move(cursor), std::move(result)};
 }
 
+void waitForTaskCompletion(exec::Task* task) {
+  if (not task->isFinished()) {
+    // The Task can return results before the Driver is finished executing.
```

Comment:
Would it make sense to move this comment to the function documentation in the .h file?

Reply:
yes! done.